### PR TITLE
fix for pagination in a directory different that /

### PR DESCRIPTION
**Pull request by tasken**

### CODE REVIEW

**Comment by parkr** [comment]
Ah! Good catch. :+1:

File: site/_docs/pagination.md [hunk 1]
```diff
@@ -204,7 +204,7 @@ page with links to all but the current page.
     {% if page == paginator.page %}
       <em>{{ page }}</em>
     {% elsif page == 1 %}
-      <a href="{{ '/index.html' | prepend: site.baseurl | replace: '//', '/' }}">{{ page }}</a>
+      <a href="{{ paginator.previous_page_path | prepend: site.baseurl | replace: '//', '/' }}">{{ page }}</a>
```
Comment (parkr) [inline site/_docs/pagination.md]: :+1:

**Comment by tasken** [comment]
First pull request on GitHub, should I do something else?

**Comment by envygeeks** [comment]
:shipit:

**Comment by tasken** [comment]
:)

**Comment by parkr** [comment]
Well done, @tasken! Hope you contribute more on GitHub. Let me know if you have any questions!

**Comment by envygeeks** [comment]
:heart:

**Comment by tasken** [comment]
Thanks for the warm welcome guys! Will do my best on future findings. Regards.

**Pull request merged**